packages/ui-components/src/components/PageHeader/PageHeader.component.tsx

```diff
@@ -58,6 +85,7 @@ export interface PageHeaderProps {
   heading?: string | React.ReactNode | null
```

Comment:
The storybook control table displays `any` here instead of the correct types. When you click the "Set object" button this results in a runtime error.